os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -845,7 +845,19 @@ void cmd_wifi_info(int argc, char **argv)
  * Wi-Fi manager does it.
  */
 #include <net/if.h>
+#ifdef CONFIG_NET_NETMGR
+FAR struct netif *netdev_findbyname(const char *ifname)
+{

```

Comment:
how about this?
```
#ifdef CONFIG_NET_NETMGR
FAR struct netif *netdev_findbyname(const char *ifname)
{
	struct netif *dev = NULL;
	if (ifname) {
		dev = netif_find((char *)ifname);
	}
	return dev;
}
```
just my opinion

Reply:
Thank you.
I think it is okay to keep current way, since I actually copy this from netdev_findbyname.c. So maybe now we keep the same.

Reply:
yes!